src/Tickets/Order_Modifiers/Checkout/Gateway/Paypal/Fees.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Fees Class for handling fee logic in the PayPal checkout process.
+ *
+ * This class manages fees, calculates them, and appends them to the cart
+ * during the PayPal checkout process. It integrates with various filters
+ * and hooks specific to PayPal's order and payment flow.
+ *
+ * @since TBD
+ * @package TEC\Tickets\Order_Modifiers\Checkout\Gateway\Paypal
+ */
+
+namespace TEC\Tickets\Order_Modifiers\Checkout\Gateway\Paypal;
```

Comment:
I would like to avoid adding PayPal code outside of the PayPal Gateway folder. Move this to a `Tickets\Commerce\Gateways\Paypal\Order_Modifiers`